packages/sit-onyx/src/components/OnyxTable/OnyxTable.vue

```diff
@@ -159,6 +205,16 @@ $border: var(--onyx-1px-in-rem) solid var(--onyx-color-base-neutral-300);
     contain: paint;
     width: 100%;
 
+    &__empty {
+      &-content {
+        display: flex;
+        justify-content: center;
+      }
+      .onyx-icon {
+        color: var(--onyx-color-text-icons-neutral-medium);
+      }
```

Comment:
Shouldn't we stick to the default `color="neutral"` property of the icon to be uniform here? Also has better contrast in my opinion :)

Reply:
needs to be answered by @jannick-ux , I was sticking to his design.

Reply:
I asked for feedback in [our teams channel](https://teams.microsoft.com/l/message/19:8500e48ce8d14968a18d1bd2d0dd293e@thread.tacv2/1719592845329?tenantId=d04f4717-5a6e-4b98-b3f9-6918e0385f4c&groupId=969ca3b3-9c78-411f-8629-eb770bbfdc7b&parentMessageId=1719592845329&teamName=onyx&channelName=Design&createdTime=1719592845329)

Reply:
martin says to use the default color. I'll update the screenshots.